server/server.go

```diff
@@ -510,7 +505,10 @@ func New(config *Config) (*Server, error) {
 		return nil, err
 	}
 
-	containerServer := libkpod.New(r, store, imageService, config.SignaturePolicyPath)
+	containerServer, err := libkpod.New(libkpod.DefaultConfig())
+	if err != nil {
```

Comment:
Maybe I'm missing it, but don't you need to get the values of r, store, imageService and config.SignaturePolicyPath into the containerServer still?  Will the DefaultConfig() always fill the bill here?  And if not, do you still need to set them in this function?